### ISSUE TITLE
[NBug] Error reading the D:\Audio\CD4\.git directory.

### ISSUE THREAD

**Issue by i4004** [gitextensions/gitextensions#7939]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.IO.FileNotFoundException: Error reading the D:\Audio\CD4\.git directory.
   at System.IO.FileSystemWatcher.StartRaisingEvents()
   at GitCommands.Settings.FileSettingsCache..ctor(String settingsFilePath, Boolean autoSave)
   at GitCommands.Settings.GitExtSettingsCache.<>c__DisplayClass2_0.<FromCache>b__0()
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.LazyInitValue()
   at GitCommands.Utils.WeakRefCache.Get[T](String objectUniqueKey, Lazy`1 provideObject)
   at GitCommands.Settings.RepoDistSettings.CreateLocal(GitModule module, RepoDistSettings lowerPriority, SettingLevel settingLevel, Boolean allowCache)
   at GitCommands.GitModule.get_EffectiveSettings()
   at ResourceManager.GitPluginBase.Register(IGitUICommands gitUiCommands)
   at System.Linq.LinqExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action)
   at GitUI.PluginRegistry.Register(IGitUICommands gitUiCommands)
   at GitUI.CommandsDialogs.FormBrowse.RegisterPlugins()
   at GitUI.CommandsDialogs.FormBrowse.<<-ctor>b__56_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Additional information
1. The directory is opening successfylly with right clicking on folder and "GitExt Open Repository"
2. When opening the same directory inside application with "Start->Open" then i am getting this error

## Environment
- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.26.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4150.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
First time this exception reported, there have been other variants of permission exceptions, corrected in 3.1-3.3.1

In this case the .git directory obviously exists, but when started without privileges, the directory cannot be read.
FileSystemWatcher creates the exeption but also if try-catch is used (done in some sitauations enabling), likely something else will fail instead.

**Comment by i4004** [comment]
I launched the app with admin privileges, and it is working now! thanks :)

2. With right click operation, it was working because I was launching it from Unreal Commander which is actually running as admin.
